src/lib.rs

```diff
@@ -483,9 +483,9 @@ struct ExpandedNameOwned<'input> {
     name: &'input str,
 }
 
-impl<'input> ExpandedNameOwned<'input> {
+impl<'a, 'input> ExpandedNameOwned<'input> {
```

Comment:
Shouldn't it be `'a, 'input: 'a`?

Reply:
I think it would, from a logical point of view, make sense that `'input` lives at least as long as `'a`, but TBH, I don't know the benefit of adding it if both versions (with and without) compile?